x-pack/plugins/security_solution/public/common/components/event_details/cti_details/translations.ts

```diff
@@ -81,19 +74,11 @@ export const NO_ENRICHMENTS_FOUND_TITLE = i18n.translate(
   }
 );
 
-export const NO_ENRICHMENTS_FOUND_DESCRIPTION = i18n.translate(
-  'xpack.securitySolution.alertDetails.noEnrichmentsFoundDescription',
-  {
-    defaultMessage:
-      'We did not find threat intelligence that matches any of the indicator match rules, or any enrichment for this alert.',
-  }
-);
-
 export const IF_CTI_NOT_ENABLED = i18n.translate(
   'xpack.securitySolution.alertDetails.ifCtiNotEnabled',
   {
     defaultMessage:
-      "If you haven't enabled any threat intelligence sources and want to learn more about this capability, ",
+      "We haven’t found field value matched a threat intelligence indicator with a rule you created. If you haven't enabled any threat intelligence sources and want to learn more about this capability, ",
```

Comment:
Is this copy still meant to capture the situation where CTI isn't enabled? If so, I don't think this additional copy is correct. See my above comment.

Reply:
To be clear: I think we should delete this addition and stick to using `NO_ENRICHMENTS_FOUND_DESCRIPTION` in tandem as per my above comment 👍

Reply:
I think I finally understand - thanks for clarifying! This copy is currently the same as the designs - but I agree that there is room for clarification. I am bringing back NO_ENRICHMENTS_FOUND_DESCRIPTION for now, but I believe @shimonmodi wanted to have some sort of a doclink in here, possibly about not enabling threat intel module but about the indicator match rule. I will defer this to @yiyangliu9286, we can make a change after this PR